ydb/core/base/ut/table_index_ut.cpp

```diff
@@ -73,6 +67,12 @@ Y_UNIT_TEST_SUITE (TableIndex) {
 
         UNIT_ASSERT(!IsCompatibleIndex(type, Table, {{"DATA1"}, {"DATA3", "DATA3"}}, explain));
         UNIT_ASSERT(!explain.empty());
+
+        UNIT_ASSERT(!IsCompatibleIndex(type, Table, {{}, {}}, explain));
+        UNIT_ASSERT(!explain.empty());
```

Comment:
I would suggest to add checks on `explain` text (in full or by a substring).

With verification methods doing checks for multiple error conditions there is always a chance that diagnostics could be mixed up (due to overlapping error conditions, for example).

Reply:
Fixed